Polyhedron/demo/Polyhedron/Plugins/PCA/Scene_aff_transformed_point_set_item.h

```diff
@@ -0,0 +1,95 @@
+#ifndef SCENE_AFF_TRANSFORMED_POINT_SET_ITEM_H
+#define SCENE_AFF_TRANSFORMED_POINT_SET_ITEM_H
+
+#if defined( scene_aff_transformed_item_EXPORTS)
```

Comment:
```suggestion
#if defined( scene_aff_transformed_point_set_item_EXPORTS)
```
Otherwise the export/import feature cannot work correctly. It has to be the name of the library defined in the `CMakeLists.txt`.